sites/bizbash/components/image-slider.marko

```diff
@@ -0,0 +1,25 @@
+import { buildImgixUrl } from '@base-cms/image';
+import { getAsArray } from '@base-cms/object-path';
+
+$ const connection = getAsArray(input, 'images.edges');
```

Comment:
`$ const images = getAsArray(input, 'images')` (see the suppliers.marko template for more info)